source/cloud-security/amazon/services/supported-services/custom-buckets.rst

```diff
@@ -0,0 +1,133 @@
+.. Copyright (C) 2015, Wazuh, Inc.
+
+.. meta::
+  :description: Learn how to configure Amazon Custom Logs Buckets fetching.
+
+.. _amazon_custom_logs:
+
+Custom Logs Buckets
+===================
+
+.. versionadded:: 4.7.0
+
+`Amazon Simple Queue Service (Amazon SQS)  <https://aws.amazon.com/sqs/>`_ is a fully managed message queuing service that offers secure, durable, and available hosted queues to decouple and scale software systems and components.
+It allows sending, storing, and receiving messages between software components at any volume, without losing messages or requiring other services to be available. These features make it an optimal component to associate with Amazon S3 Buckets to consume any type of log.
+
+Combining Amazon SQS with Amazon S3 buckets, allows Wazuh to fetch any JSON, CSV or plain text logs, originated in AWS or not, and process the events inside them.
```

Comment:
The difference with what already exists is the following:
- Currently, all Wazuh's AWS integrations to read logs stored in buckets like Cloudtrail, Macie, etc. they require specifying the path in the configuration and that the logs are saved in a defined and very strict folder structure. These structures are the ones that can be consulted [here (Paths to logs)](https://documentation.wazuh.com/current/cloud-security/amazon/services/supported-services/index.html). Wazuh directly accessed those paths and downloaded the items found inside.
- Now, this new feature of `Custom Logs Buckets` allows to read logs stored in any path of a bucket, even from multiple buckets. In addition, it allows one to do it in a distributed and simultaneous way with different managers or Wazuh agents, without duplicating logs. This is done because the bucket publishes a notification to the SQS queue containing the full path of each new file stored in it.